docs/userguide.md

```diff
@@ -710,18 +710,17 @@ experimenting with KubeFed on a production cluster.
 
 ### Helm Configuration
 
-To deploy a federation in a namespaced configuration, set
+To deploy KubeFed in a namespaced configuration, set
 `global.scope` to `Namespaced` as per the Helm chart [install
 instructions](https://github.com/kubernetes-sigs/kubefed/blob/master/charts/kubefed/README.md#configuration).
 
+### Joining additional clusters
 
-### Joining Clusters
-
-Joining additional clusters to a namespaced federation requires
+Joining additional clusters to a namespaced deployment requires
 providing additional arguments to `kubefedctl join`:
 
 - `--kubefed-namespace=<namespace>` to ensure the cluster is joined
-  to the federation running in the specified namespace
+  to the federated deployment running in the specified namespace
```

Comment:
Small-f 'federation' is appropriate here, and refers to 'a set of clusters being managed by KubeFed'.  Please revert.